artemis/config_templates/brokered/broker.xml

```diff
@@ -115,6 +115,8 @@ under the License.
       <acceptors>
          <acceptor name="amqp">tcp://0.0.0.0:5672?protocols=AMQP;saslMechanisms=PLAIN</acceptor>
          <acceptor name="amqps">tcp://0.0.0.0:5671?protocols=AMQP;sslEnabled=true;saslMechanisms=PLAIN;keyStorePath=${EXTERNAL_KEYSTORE_PATH};keyStorePassword=enmasse</acceptor>
+         <acceptor name="openwire">tcp://0.0.0.0:61613?protocols=OPENWIRE;sslEnabled=true;saslMechanisms=PLAIN;keyStorePath=${EXTERNAL_KEYSTORE_PATH};keyStorePassword=enmasse</acceptor>
+         <acceptor name="core">tcp://0.0.0.0:61616?protocols=CORE;sslEnabled=true;saslMechanisms=PLAIN;keyStorePath=${EXTERNAL_KEYSTORE_PATH};keyStorePassword=enmasse</acceptor>
```

Comment:
I don't think saslMechanisms=PLAIN is required / makes any sense for the non-AMQP protocols.  They only support username/password style authentication.

Reply:
Fixed